vcpkg/consume/gha-authentication.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Tutorial: Authenticate vcpkg Git-based registries in GitHub Actions"
+description: Learn to configure Git credentials for private repositories when using vcpkg on GitHub Actions
+author: vicroms
+ms.author: viromer
+ms.date: 9/21/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+#CustomerIntent: As an enterprise vcpkg user I want to authenticate to private Git registries so that I can use them as Git-based repositories
+---
+# Tutorial: Authenticate vcpkg Git-based registries in GitHub Actions
+
+When using vcpkg with private registries it is necessary to provide access
+credentials to authenticate Git operations executed internally by vcpkg.
+
+In this tutorial you will learn how to authenticate a private Git repository to use in your GitHub
+Actions workflow.
+
+> [!div class="checklist"]
+> * [Create a Personal Access Token for your private repository](#1---create-a-personal-access-token-for-your-private-repository)
+> * [Create a GitHub Actions secret to use in your workflow](#2---create-a-github-actions-secret-to-use-in-your-workflow)
+> * [Authenticate to private Git repositories in your workflow](#3---authenticate-to-private-git-repositories-in-your-workflow)
+
+
+## Prerequisites
+
+* A GitHub account
+* A private vcpkg Git-registry hosted on GitHub
+* vcpkg
+
+## 1 - Create a Personal Access Token for your private repository
+
+Follow the [instructions on GitHub to generate a fine-grained Personal Access Token
+(PAT)](<https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token>)
+and grant it permission to access the private repository where your vcpkg Git registry is hosted.
+
+
+## 2 - Create a GitHub Actions secret to use in your workflow
+
+> [!NOTE]
+> Never store a GitHub Personal Access Token in your repository files
+
+The next step is to follow [GitHub's instructions to store the PAT as a secret available in your
+GitHub Actions
+workflows](<https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository>).
+
+### 3 - Authenticate to private Git repositories in your workflow
```

Comment:
Hmm I don't think this one should be a different indentation level than the others if they are in the same numbered series?